templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,18 @@
 # Press Enter to send
 
-By default, Zulip sends your message when you press enter (or return)
+By default, Zulip sends your message when you press **enter** (or **return**)
 on your keyboard. If you want to turn this off, simply uncheck the
 **Press Enter to send** box below the compose field:
 
 ![Box checked](/static/images/help/enter-box-checked.png)
 
-You will now see a **Send** button below the compose field looking
-like this.
+You will now see a **Send** button below the compose field like this:
 
 ![Box unchecked](/static/images/help/enter-box-unchecked.png)
 
-With the box unchecked, you will have to click the **Send** button to
-send your message. Pressing enter will no longer send your message.
+With the box unchecked, you will have to manually click the **Send** button to
+send your message. Pressing **enter** will no longer send your message.
 
-If you’d like to re-enable the press enter to send feature, just check
-the **Press Enter to send** box again, and you will be able to send
-your messages by pressing enter.
-
-**[Go to "Table of contents"](/help/#using-zulip)**
+If you’d like to re-enable the press **enter** to send feature, just check
+the **Press Enter to send box** again, and you will be able to send
+your messages by pressing the **enter** key.
```

Comment:
Can you convert the doc to numerical step form (1. 2. 3.)? I think there's enough parts to do so